.github/actions/test-visionos-rntester/action.yml

```diff
@@ -38,6 +38,14 @@ runs:
         ruby-version: ${{ inputs.ruby-version }}
     - name: Setup xcode build cache
       uses: ./.github/actions/setup-xcode-build-cache
+    - name: Download visionOS SDK
+      run: |
+        # https://github.com/actions/runner-images/issues/10559
+        sudo xcodebuild -runFirstLaunch
+        sudo xcrun simctl list
+        sudo xcodebuild -downloadPlatform visionOS
```

Comment:
any chance this can be cached? downloading 6GB on every run is gonna add a few minutes to the CI time

Reply:
I can try adding the cache action, but we may run up against the storage limits for the action: https://github.com/actions/cache?tab=readme-ov-file#cache-limits. It looks like this repo is already [over the limit](https://github.com/callstack/react-native-visionos/actions/caches), which could cause problems if more expensive artifacts get evicted from the cache.

An alternative would be to switch to macos-15/macos-latest for builds, which do have the visionos image, but I don't know if builds work on those runners.